apps/wallet-mobile/src/TxHistory/TxHistoryList/TxHistoryList.tsx

```diff
@@ -109,8 +110,7 @@ const styles = StyleSheet.create({
     flex: 1,
   },
   dayHeaderRoot: {
-    paddingTop: 16,
-    paddingBottom: 8,
+    paddingBottom: 4,
```

Comment:
paddingTop here was causing the first row to add 16 + 8 (spacer) between the locked deposit and tx history